src/tools/fuzzing.h

```diff
@@ -1728,12 +1735,18 @@ class TranslateToFuzzReader {
   }
 
   Expression* makeConst(Type type) {
-    if (type == exnref) {
-      // There's no exnref.const.
-      // TODO We should return a nullref once we implement instructions for
-      // reference types proposal.
-      assert(false && "exnref const is not implemented yet");
+    switch (type) {
+      case anyref:
+        assert(false && "anyref const is not implemented yet");
+      case exnref:
+        // There's no exnref.const.
+        // TODO We should return a nullref once we implement instructions for
+        // reference types proposal.
+        assert(false && "exnref const is not implemented yet");
```

Comment:
We can probably put the TODO on top both reference types, because it applies to all reference types